### PR TITLE
Change Vocabulary Logo in Sidebar and Readme_assets

### PR DESCRIPTION
**Pull request by jahnvigupta**
## Fixes

Related to #408 by @dhruvkb 
## Description

This PR updates the Vocabulary Logo in Sidebar and in Readme_assets.
## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by jahnvigupta** [comment]
@dhruvkb I have updated the logo in Sidebar and Readme Assets folder but couldn't understand which images you meant by ` twitter:image` and `og:image`.

File: .storybook/theme.js [hunk 1]
```diff
@@ -27,5 +27,5 @@ export default create({
   // Branding
   brandTitle: 'Vocabulary',
   brandUrl: 'https://opensource.creativecommons.org/cc-vocabulary',
-  brandImage: 'https://raw.githubusercontent.com/creativecommons/vocabulary/master/readme_assets/vocabulary_logo.svg?sanitize=true'
+  brandImage: 'logos/products/vocabulary.svg#vocabulary'
```
Comment (dhruvkb) [inline .storybook/theme.js:30]: This is nice.

**Review by dhruvkb — APPROVED** [review]
You 👍-ed @kushaggarwal's [comment](https://github.com/creativecommons/vocabulary/issues/408#issuecomment-633660643) asking to take `README.md` and then you changed the logo in `readme_assets/`? That's not 100% fair to him.

**Comment by dhruvkb** [comment]
@kushaggarwal I'll merge this PR once you update the dimensions in the `README.md` file to match the new logo.

**Comment by jahnvigupta** [comment]
@kushaggarwal , @dhruvkb I am sorry, actually changing the logo in readme_assets didn't change it in readme file, so I thought it to be separate.

**Comment by dhruvkb** [comment]
That's ok, @kushaggarwal still needs to make the corresponding changes in the `README.md` file.

**Comment by kushaggarwal** [comment]
No problem @jahnvigupta 
@dhruvkb will update and push the  changes for the readme 
Thank you

**Comment by dhruvkb** [comment]
Actually it's not blocked because the README.md file uses `` tags instead of `<svg>`.

**Pull request merged**